django_rq/workers.py

```diff
@@ -23,3 +23,20 @@ def get_worker(*queue_names):
     return Worker(queues,
                   connection=queues[0].connection,
                   exception_handlers=get_exception_handlers() or None)
+
+
+def collect_workers_per_configuration(queue_list):
```

Comment:
If I understood it well, `CONFIG_1` and `CONFIG_2` are dictionaries, and when I do 
```python
{
    CONFIG_1: [worker1],
    CONFIG_2: [worker2],
}
```
What I am really doing is using a dict as a hash to another dict. Well, it gives us a `TypeError`, because dict is an *unhashable type*.

Let me know if did not understand it well.

In relation to the names, I agree with you, but I will wait to submit when this thread close.